src/routes/event-match.tsx

```diff
@@ -39,6 +44,35 @@ const blueStyle = css``
 // We are using margins instead of grid-gap
 // because we don't want grid-gap to be applied to the empty spacing columns
 
+const matchStyle = css`
+  /* extra selectors for specificity */
+  a.${tableTeamStyle}.${redStyle} {
+    color: ${red};
+  }
+
+  a.${tableTeamStyle}.${blueStyle} {
+    color: ${blue};
+  }
+`
+
+const offlineMatchStyle = css`
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-areas:
+    'topArea'
+    'leftColumn';
+  padding: 0.75rem;
+  @media (max-width: 930px) {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      'topArea'
+      'leftColumn';
```

Comment:
This grid-template-areas property is the same as the one outside of the media query, so there is no need to override it here.